test/unittest/compiler/unittest_realizer.cpp

```diff
@@ -95,103 +103,29 @@ TEST(RecurrentRealizer, recurrent_return_sequence_single_p) {
     {"fully_connected", {"name=fc_out", "input_layers=lstm"}}};
 
   std::vector<LayerRepresentation> expected = {
+    /// t - 0
     {"lstm",
      {"name=lstm/0", "input_layers=source", "max_timestep=3", "timestep=0"}},
     {"fully_connected", {"name=fc_out/0", "input_layers=lstm/0"}},
+
+    /// t - 1
     {"lstm",
      {"name=lstm/1", "input_layers=fc_out/0", "shared_from=lstm/0",
       "max_timestep=3", "timestep=1"}},
     {"fully_connected",
      {"name=fc_out/1", "input_layers=lstm/1", "shared_from=fc_out/0"}},
+
+    /// t - 2
     {"lstm",
      {"name=lstm/2", "input_layers=fc_out/1", "shared_from=lstm/0",
       "max_timestep=3", "timestep=2"}},
     {"fully_connected",
      {"name=fc_out/2", "input_layers=lstm/2", "shared_from=fc_out/0"}},
-    {"concat", {"name=fc_out", "input_layers=fc_out/0,fc_out/1,fc_out/2"}},
-  };
-
-  realizeAndEqual(r, before, expected);
-}
-
-TEST(RecurrentRealizer, recurrent_multi_inout_p) {
```

Comment:
This test is not an intended use case (it's too lengthy because of dummy layers and now we don't need this so we don't have to test for this case) so removed for brevity.